src/Exception/FormatTranslationException.php

```diff
@@ -16,19 +18,24 @@ class FormatTranslationException extends AbstractFormatTranslationException impl
      *
      * @see \Exception::__construct()
      * @since 0.1
+     *
+     * @param string|Stringable|null   $format     The format string being translated, if any.
+     * @param TranslatorInterface|null $translator The translator performing the translation, if any.
+     * @param mixed|Value              $context    The context of translation, if any.
+     * @param array|null               $params     The interpolation parameters used, if any.
      */
     public function __construct(
             $message = '',
             $code = 0,
             \Exception $previous = null,
-            $subject = null,
+            $format = null,
             TranslatorInterface $translator = null,
             $context = null,
             $params = null)
```

Comment:
Should the indentation here be as follows?

```
...
    $content = null
    $params = null
) {
    // code
...
```

Reference: [PSR-2: Method Arguments](http://www.php-fig.org/psr/psr-2/#method-arguments)

Same goes for the other constructors.